<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-R4ML4PD" name="Test" tests="4" failures="2" timestamp="4 Jan 2017 07:18:58 GMT" time="66.229" errors="0">
  <testcase name="TC_1" time="0.123" classname="com.mediastep.test.DemoTestlink">
    <failure type="java.lang.AssertionError" message="TC1 was failed:  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: TC1 was failed:  expected [true] but found [false]
	at com.mediastep.test.DemoTestlink.TC_1(DemoTestlink.java:30)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TC_1 -->
  <testcase name="TC_2" time="3.904" classname="com.mediastep.test.DemoTestlink">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.mediastep.test.DemoTestlink.TC_2(DemoTestlink.java:63)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- TC_2 -->
  <testcase name="TC_3" time="7.195" classname="com.mediastep.test.DemoTestlink"/>
  <testcase name="TC_4" time="3.563" classname="com.mediastep.test.apitest.Demoapitest"/>
</testsuite> <!-- Test -->
